docs/locales/ko/LC_MESSAGES/manager/user-api/vfolders.po

```diff
@@ -1767,7 +1767,7 @@ msgstr "``target_name``"
 
 #: ../../manager/user-api/vfolders.rst:1312 71559885111249d8966e3abc512fda0e
 msgid "The name of the new virtual folder."
-msgstr "새 가상 폴더의 이름"
+msgstr "새 가상 폴더의 이름."
```

Comment:
Since the original text is not a sentence, it is correct to remove the dot from both the original text and the translation.